README.md

```diff
@@ -127,6 +127,7 @@ A sample configuration file with all options is available [here](https://github.
   * `"arguments"` checks alignment of function call arguments.
   * `"statements"` checks alignment of statements.
 * `ban` bans the use of specific functions. Options are ["object", "function"] pairs that ban the use of object.function()
+* `class-member-no-default-access` enforces using explicit visibility on class members
```

Comment:
this is too verbose. let's call this `member-access`. in general, we should be limiting rule name verbosity.